pkg/smi/types.go

```diff
@@ -94,3 +77,21 @@ func WithKubeController(c k8s.Controller) TrafficSplitListOption {
 		o.KubeController = c
 	}
 }
+
+// GetSmiClientVersionHTTPHandler returns an http handler that returns supported smi version information
+func GetSmiClientVersionHTTPHandler() http.Handler {
```

Comment:
@shashankram do we still need this? Can we just recommend users inspect the CRDs?